third_party/2and3/datetimerange/__init__.pyi

```diff
@@ -0,0 +1,38 @@
+from typing import Any, Optional, Union, Iterable
+import datetime
+from dateutil.relativedelta import relativedelta
+
+class DateTimeRange:
```

Comment:
Since this stub is for Python 2 as well, this should use `object` as base class to mark this as a new-style class:

```python
class DateTimeRange(object):
```